catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -41,6 +40,89 @@ function wrapNum(x, range) {
     return ((x - min) % d + d) % d + min;
 }
 
+function findExtreme({objArray, property, comparator}) {
+    if (objArray.length === 0) {
+        return undefined;
+    }
+    return objArray.reduce((extreme, coordinateObj) => {
+        extreme = comparator(extreme, coordinateObj[property]);
```

Comment:
Do we need to assign the result before returning it? Also, if we remove the assignment it would be a single line, and we could simply do:

`(extreme, coordinateObj) => comparator(extreme, coordinateObj[property])`